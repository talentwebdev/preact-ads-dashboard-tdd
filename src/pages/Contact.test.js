import { shallow } from 'enzyme';
import Contact from './Contact';

describe('Contact', () => {
    let contactContainer;

    beforeEach(() => contactContainer = shallow(<Contact/>));

    it('should render the div element', () => {
        expect(contactContainer.find('div').length).toBeGreaterThanOrEqual(1);
    });
});