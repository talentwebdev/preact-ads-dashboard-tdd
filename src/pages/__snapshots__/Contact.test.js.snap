// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact renders Contact correctly 1`] = `ShallowWrapper {}`;

exports[`Contact renders Contact correctly: div 1`] = `ShallowWrapper {}`;
